src/operator/pad-inl.h

```diff
@@ -88,8 +86,8 @@ class PadOp : public Operator {
           out_data[pad_enum::kOut].get<xpu, 5, DType>(s);
       pad_image(out, data, param_.pad_width, param_.mode, constant_value);
     } else {
-      LOG(FATAL) << "Only 4d or 5d input tensors with padding applied to "
-                    "dimensions > 1 is currently implemented.";
+      LOG(FATAL) << "Current Implementation only supports 4-D or 5-D input arrays with padding applied "
+                    "only on the lower 2 or 3 dimensions(expects higher 2 dimensions in pad_width to be zero).";
```

Comment:
Could you be more specific than "lower 2 or 3 dimensions". I'm not sure it's clear to the reader. Also I think this is may muddle the use of "dimensions" vs "axis" in the way that we're trying to avoid.

You could say "with padding applied only on axes 1, 2, and 3. Expects axes 4 and 5 in `pad_width` to be zero"?. 

Also, note pad_width should be in backticks.